source/elements/oneMKL/source/domains/blas/trmm.rst

```diff
@@ -328,5 +478,136 @@ trmm (USM Version)
 
    :ref:`oneapi::mkl::unimplemented<onemkl_exception_unimplemented>`
       
+**Out-of-place API**
+--------------------
+
+.. rubric:: Syntax
+
+.. code-block:: cpp
+
+   namespace oneapi::mkl::blas::column_major {
+       void trmm(sycl::queue &queue,
+                 onemkl::side left_right,
+                 onemkl::uplo upper_lower,
+                 onemkl::transpose trans,
+                 onemkl::diag unit_diag,
+                 std::int64_t m,
+                 std::int64_t n,
+                 value_or_pointer<T> alpha,
+                 const T *a,
+                 std::int64_t lda,
+                 const T *b,
+                 std::int64_t ldb,
+                 value_or_pointer<T> beta,
+                 T *c,
+                 std::int64_t ldc,
+                 const std::vector<sycl::event> &dependencies = {})
+   }
+
+.. code-block:: cpp
+
+   namespace oneapi::mkl::blas::row_major {
+       void trmm(sycl::queue &queue,
+                 onemkl::side left_right,
+                 onemkl::uplo upper_lower,
+                 onemkl::transpose trans,
+                 onemkl::diag unit_diag,
+                 std::int64_t m,
+                 std::int64_t n,
+                 value_or_pointer<T> alpha,
+                 const T *a,
+                 std::int64_t lda,
+                 const T *b,
+                 std::int64_t ldb,
+                 value_or_pointer<T> beta,
+                 T *c,
+                 std::int64_t ldc,
+                 const std::vector<sycl::event> &dependencies = {})
+   }
+
+
+.. container:: section
+
+   .. rubric:: Input Parameters
+
+queue
```

Comment:
Did you check how these look after compiling? Indentation does not seem to match to in-place API descriptions.